src/app_builder.rs

```diff
@@ -564,14 +564,14 @@ where
             gov: self.gov,
             stargate: self.stargate,
         };
-
-        let mut app = App {
+        let api = self.api;
+        let mut storage = self.storage;
+        init_fn(&mut router, &api, &mut storage);
+        App {
             router,
-            api: self.api,
+            api,
             block: self.block,
-            storage: self.storage,
-        };
-        app.init_modules(init_fn);
-        app
+            storage,
+        }
```

Comment:
Is there a reason to change this? I think the original looked a bit nicer.

Reply:
Yeah, made it look nicer. Original reason was to get rid of calling `app.init_modules`, because this function is used only in `AppBuilder`, but this is out of scope of current change.